src/app/submission/Submission.ts

```diff
@@ -63,7 +63,7 @@ export class Submission {
       throw Error('Payments are not handled by this class');
     }
     this.bsn = this.parsedSubmission.bsn;
-    this.kvk = this.parsedSubmission.kvk;
+    this.kvk = this.parsedSubmission.kvknummer;
```

Comment:
Dit moeten we even checken, Marnix heeft dit gewijzigd maar weet even niet waarom :). Kan dingen stuk maken.

Reply:
Gecheckt, deze fix mag mee